### DISCUSSION TITLE
First impression

### DISCUSSION THREAD

**Discussion by eltikia** [eltikia/Perl-1.0#52] [Q&A] [answered]
Can you give us your first impression of the Perl 1 compiler?

**Comment by Stagyrite** [comment, marked as answer]
My first impression would be that what's in Perl 1 is often bogus. It's from the year 1987, and therefore, you must be careful about using it anyhow. Numerical operations can lead to an unexpected error nowadays. Perl 1 isn't written in Perl, contrary to new versions of Perl that are written in Perl. One should know translation methods to understand it better and dig into what [YACC](https://en.wikipedia.org/wiki/Yacc "Yacc - Wikipedia") does. The code would give a pretty good impression to somebody skilled in Perl programming and compilers.
